src/c/owner-message-content.js

```diff
@@ -0,0 +1,105 @@
+                to_user_id: h.getCurrentProject()['project_user_id']
+              });
+              l = m.postgrest.loaderWithToken(loaderOpts);
+
+              l.load().then(sendSuccess(true));
+          };
+
+          return {
+            sendMessage: sendMessage,
+            sendSuccess: sendSuccess,
+            userDetails: args
+          };
+        },
+        view: (ctrl, args) => {
+          const user = h.getUser(),
+           successMessage = m(".modal-dialog-content.u-text-center",
+              [
+                m(".fa.fa-check-circle.fa-5x.text-success.u-marginbottom-40"),
+                m("p.fontsize-large", 
+                  `Sua mensagem foi enviada com sucesso para ${_.first(ctrl.userDetails()).name}. Você vai receber uma cópia no seu email e pode seguir a conversa por lá!`
+                )
+              ]
+            ),
+            contactForm = [
+              m('.modal-dialog-content',
+                  [
+                    m('.w-form',
+                      [
+                        m('form',
+                          [
+                            m('.w-row',
+                              [
+                                m('.w-col.w-col-6',
+                                  [
+                                    m('label.fontsize-smaller',
+                                    'Seu nome'
+                                    ),
+                                    m(`input.w-input.text-field[value='${user ? user['name'] : ''}'][id='from_name'][type='text'][required]`)
+                                  ]
+                                ),
+                                m('.w-col.w-col-6',
+                                  [
+                                    m('label.fontsize-smaller',
+                                    'Seu email'
+                                    ),
+                                    m(`input.w-input.text-field[value='${user ? user['email'] : ''}'][id='from_email'][type='text'][required]`)
+                                  ]
+                                )
+                              ]
+                            ),
+                            m('label',
+                            'Mensagem'
+                            ),
+                            m('textarea.w-input.text-field.height-small[id=\'message_content\'][required]')
+                          ]
+                        )
+                      ]
+                    ),
+                    m('.fontsize-smallest.fontcolor-terciary',
+                    'Você receberá uma cópia desta mensagem em seu email.'
+                    )
+                  ]
+                ),
+              m('.modal-dialog-nav-bottom',
+                m('.w-row',
+                    [
+                      m('.w-col.w-col-3'),
+                      m('.w-col.w-col-6',
+                      m('a.btn.btn-large[href=\'js:void(0);\']',{onclick: ctrl.sendMessage },
```

Comment:
Also, we should make this element a `button` and handle the form submit on the form `onsubmit` event rather than here.